sql/2022/interop/align_items_values.sql

```diff
@@ -0,0 +1,70 @@
+#standardSQL
+# Popularity of CSS align values
+CREATE TEMPORARY FUNCTION getAlignItems(css STRING)
+RETURNS ARRAY<STRING> LANGUAGE js
+OPTIONS (library = "gs://httparchive/lib/css-utils.js") AS '''
+try {
+  var reduceValues = (values, rule) => {
+    if ("rules" in rule) {
+      return rule.rules.reduce(reduceValues, values);
+    }
+    if (!("declarations" in rule)) {
+      return values;
+    }
+
+    var alignItems = rule.declarations.find(
+      (d) => d.property == "align-items"
+    );
+
+    if (!alignItems) {
+      return values;
+    }
+
+    return values.concat(alignItems.value.replace("!important", "").split(" ").filter(Boolean));
```

Comment:
Do we want to `split` for `align-items`?

We could have values such as `align-items: first baseline;`

Reply:
Yes good call